aten/src/ATen/function_wrapper.py

```diff
@@ -119,6 +119,18 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 .registerOp<${return_type} (${formals_types})>(Backend::${Backend}, "${schema_string}", &${Type}::${api_name})
 """)
 
+C10_DEFAULT_FUNCTION_REGISTRATION = CodeTemplate("""\
+.op("${schema_string}", torch::RegisterOperators::options()
+  .impl_unboxedOnlyCatchAllKernel<${return_type} (${formals_types}), &TypeDefault::${api_name}>()
+  .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA))
+""")
+C10_BACKEND_FUNCTION_REGISTRATION = CodeTemplate("""\
+.op("${schema_string}", torch::RegisterOperators::options()
+  .impl_unboxedOnlyKernel<${return_type} (${formals_types}), &${Type}::${api_name}>(
+    c10::backendToTensorTypeId(Backend::${Backend}))
+  .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA))
```

Comment:
Why isn't this alias analysis setting the default?

Reply:
Because we want the custom op API to be conservative by default.